tests/unit/test_exceptions.py

```diff
@@ -79,9 +79,36 @@ def testCodeInvariants(self):
 
     def testInstantiation(self):
         for class_ in self._getExceptionClasses():
-            numInitArgs = len(inspect.getargspec(class_.__init__).args) - 1
-            args = ['arg' for _ in range(numInitArgs)]
-            instance = class_(*args)
+            # some exceptions are becoming too complicated to instantiate
```

Comment:
I actually quite liked the old idea of keeping these two in their own test, where we tested the messages of the errors that we got back in detail. To me, this test is just to make sure we instantiate correctly and that we get back a string of some sort when we as for it. So, I would take the detailed tests for the ValidationFailureExceptions out of this loop, and spin them back into their own test. Then, I would have here something like
```python
if class_ == exceptions.RequestValidationFailureException:
    args = ["{}", protocol.SearchReadsRequest()]
elif # etc
    args = #etc
else:
    numInitArgs = #etc
    args = ['arg' for _ in range(numInitArgs)]
instance = class_(*args)
# tests
```